README.md

```diff
@@ -99,6 +99,7 @@ The JavaScript npm module that implements the Orbit Chat protocol.
 There has been some work on non-JavaScript versions of OrbitDB and some of its related submodules. Most of this work has been made by ambitious, awesome members of the community who have specific needs, and so often that work is not part of this organization. The best place to find out what is available is likely by asking in the [Gitter](), but here is a short list that points to some relevant repositories. If you know of any other repos, add them here!
```

Comment:
It's confusing that we talk about how the work is often not part of thre org and then list repos in our org:

How about something like:

There has been a recent explosion of awesome, ambitions, community-driven work towards interoperability with other languages. First came the **[orbit-db-http-api](https://github.com/orbitdb/orbit-db-http-api)**, followed by **[py-orbit-db-http-client](https://github.com/orbitdb/py-orbit-db-http-client)**, and  **[go-orbit-db](https://github.com/orbitdb/go-orbit-db)**. 

These repos started as community efforts and were brought into the organization soon after. However, search GitHub - there's a lot more.

Some ambitions, awesome members of the community have created non-Javascript implementations, based on the [orbit-db-http-api](https://github.com/orbitdb/orbit-db-http-api)

Reply:
> It's confusing that we talk about how the work is often not part of thre org and then list repos in our org:

Good point.